dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -94,6 +94,14 @@ else
   _NODES_DEPLOYED = $(shell iotlab-experiment --jmespath='"0"' --format='" ".join' get $(_IOTLAB_EXP_ID) --deployment)
   _NODES_LIST_OPTION = --nodes
   _NODES_FLASH_OPTION = --flash
+  ifeq (,$(filter wsn430-%,$(BOARD)))
```

Comment:
(note from offline discussion)

Maybe combine lists, and add comment also for wsn430-.

Reply:
Addressed :)